webapp/components/wpt-results.html

```diff
@@ -446,59 +443,48 @@ <h3>
       }
 
       refreshDisplayedNodes() {
```

Comment:
This method generally looks good to me. However, I'm not very confident. If you know how to write web component tests, this is a something we definitely want to test. There's an existing fixture at `webapp/test/wpt-results.html`.